doc/man8/pbs_sched.8B

```diff
@@ -610,9 +605,9 @@ Default:
 
 .IP preempt_checkpoint 13
 .B Deprecated. 
-Add "C" to 
+Add "C" to the
```

Comment:
This isn't needed, preempt_checkpoint was part of the original NASA beta back in the late 90s.  It never made it into a real version.